services/catalog/service/src/index.ts

```diff
@@ -41,6 +43,16 @@ async function bootstrap(): Promise<void> {
 
   const { readiness } = setupLivenessAndReadiness(config);
 
+  // Check ID service is available
+  if (!(await isServiceAvailable(config.idServiceAuthBaseUrl))) {
```

Comment:
catalog-service does not need id-service to run. so no need for any check here

Reply:
ok we use RMQ messages in catalog-service, and id-service is needed to validate JWTs. we can keep this check for that